packages/rule-tester/src/RuleTester.ts

```diff
@@ -1030,6 +1050,73 @@ function assertASTDidntChange(beforeAST: unknown, afterAST: unknown): void {
   assert.deepStrictEqual(beforeAST, afterAST, 'Rule should not modify AST.');
 }
 
+/**
```

Comment:
[Refactor] The original ESLint PR had these in their own file. I think it'd be a good idea to do that here for a couple of reasons:

* Since we're essentially maintaining a long-running fork, it'll be easier to keep things in sync the more closely we mirror their structure
* This file is already pretty huge

Reply:
I completely agree, I just missed that when reviewing ESLint's PR haha